package.json

```diff
@@ -27,50 +26,53 @@
     "test:watch": "vitest --watch"
   },
   "dependencies": {
-    "@vueuse/core": "^10.11.0"
+    "@vueuse/core": "^10.11.1"
   },
   "devDependencies": {
-    "@nuxt/kit": "^3.12.2",
-    "@nuxt/schema": "^3.12.2",
-    "@nuxt/test-utils": "^3.12.2",
-    "@nuxt/vite-builder": "^3.12.2",
+    "@eslint/eslintrc": "^3.1.0",
```

Comment:
The description of this package seems to suggest that it's some sort of internal eslint package?
![image](https://github.com/user-attachments/assets/f979eb30-1fca-4a87-815a-6857294ad014)

Reply:
This is generated from: https://www.npmjs.com/package/@eslint/migrate-config because Flat config is a pain. we can improve it later if we see the need.